precompile/contracts/warp/contract_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	agoUtils "github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
```

Comment:
isn't this the same import as above?

Reply:
Removed the non-alias import in favor of the `avalancheWarp` aliased-import (as on master branch)